### PR TITLE
Multi-platform build

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,69 +30,86 @@ commands:
           command: |
             ./opt/readies/bin/getpy3
             BREW_NO_UPDATE=1 ./opt/system-setup.py
-            # git clone git://github.com/antirez/redis.git --branch 6.0.1; (cd redis; make malloc=libc -j $(nproc); make install); redis-server --version
+            # ./opt/readies/bin/getredis -v 6 --force
```
Comment (rafie) [inline .circleci/config.yml:33]: make sure Redis is built with `malloc=libc`.

**Comment by filipecosta90** [comment]
LGTM @rafie :clap: 
we just need to change the required steps names on GH correct? Who has permissions to do it (@lantiga @gkorland )?

File: .circleci/config.yml (hunk 1 above)
Comment (lantiga) [reply .circleci/config.yml:33]: Hi @rafie, looks great. I'm pretty sure `libc` is the default on macOS

**Comment by lantiga** [comment]
@gkorland I've been told you're the one doing this for other modules. Do you have time to take care of this?

**Pull request merged**